doc/en/user/source/webadmin/data/index.rst

```diff
@@ -4,15 +4,36 @@ Data
 ====
 
 
-This section is the largest and perhaps the most important section of the Web Administration Interface. Each subsection links directly to a data type page with add, edit, and delete capabilities.
+This section is the largest and perhaps the most important section of the Web Administration Interface. 
+It describes the core configuration data types that GeoServer uses to access and publish geospatial information.
+Each subsection describes a data type page which provides add, view, edit, and delete capabilities.
 
-In the example below, the data view page displays a table of indexed data. 
+.. toctree::
+   :maxdepth: 2
+
+   workspaces
+   stores
+   layers
+   layergroups
+   styles
+   
+Using Data type List views
```

Comment:
Perhaps take off the headings from here down on this page. A second-level heading here shows up in the parent TOC as the same level as workspaces stores, et al, which is probably not what we want.